test/loop/loopinversion.js

```diff
@@ -70,5 +70,10 @@ function g5()
 };
 g5();
 
+// Tests that loop inversion does not crash when the outer loop does not have
+// a condition. In order to not stay in an infinite loop, these functions return
+// immediately. The bytecode will still be generated.
+function g6(p){if(!p) return; var a = 0; for (var h = 0;; ++h){ for (var r = 0; r < 1; ++r){ a = r; }}};g4a(0);

```

Comment:
This line calls `g4a(0)`, is it meant to call `g6(0)` ?

Reply:
95c41a5f0f9c4492673c08537b7d8d65d32c2942